### DISCUSSION TITLE
Generating release apk for the new architecture throws bunch of error

### DISCUSSION THREAD

**Discussion by BraveEvidence** [reactwg/react-native-new-architecture#67] [Q&A] [answered]
I created a new `react-native` project. I created a release `apk` without enabling the new arch and it generated the `apk` correctly. 

Then I enabled the new arch and hermes

`enableHermes: true,`

`newArchEnabled=true`

and then try to generate via android studio I get following errors
<img width="1429" alt="Screenshot 2022-07-29 at 1 16 43 PM" src="https://user-images.githubusercontent.com/24708929/181716183-21e68129-0e10-4ee2-8e01-5017b276af44.png">
<img width="1440" alt="Screenshot 2022-07-29 at 1 16 50 PM" src="https://user-images.githubusercontent.com/24708929/181716202-eed6e6d6-28e4-48ba-ac4b-c4b39df6698f.png">

Later on I tried using cmd with following command

`./gradlew bundleRelease`

I get following error

<img width="1375" alt="Screenshot 2022-07-29 at 1 24 35 PM" src="https://user-images.githubusercontent.com/24708929/181716407-33f7eeee-1a2c-453e-8610-2734ff41fc24.png">

I tried cleaning and invalidating cache as well

Repo is here https://github.com/PritishSawant/ReactNativeAndroidReleaseApkNewArch

**Comment by cortinico** [comment, marked as answer]
Reference to the issue on React Native: https://github.com/facebook/react-native/issues/34304
Please check my last comment. I believe your issue is related to the lack of either the `ANDROID_HOME` or `ANDROID_SDK_ROOT` environment variable.

As a rule of thumb, you should run `react-native doctor` to make sure your environment is properly configured.

**Comment by BraveEvidence** [comment]
@cortinico  Thanks


Help for future me

https://stackoverflow.com/questions/28296237/set-android-home-environment-variable-in-mac

**Reply by leotm** [reply]
or even better our docs https://reactnative.dev/docs/environment-setup

![Capture](https://user-images.githubusercontent.com/1881059/183238431-e5994040-cec5-479e-9026-7701fcca38f6.PNG)
